src/main/java/com/google/code/twig/standard/StandardObjectDatastore.java

```diff
@@ -97,6 +97,20 @@ public TypeConverter getTypeConverter()
 		}
 		return converter;
 	}
+
+
+	public void resetTypeConverters() {
```

Comment:
Not thread safe as it may be called from constructor. 

T1 new Datastore()
T2 new Datastore()